back/engines/free/email_campaigns/spec/mailers/your_proposed_initiatives_digest_mailer_spec.rb

```diff
@@ -5,7 +5,7 @@
 RSpec.describe EmailCampaigns::YourProposedInitiativesDigestMailer do
   describe 'YourProposedInitiativesDigest' do
     let_it_be(:recipient) { create(:admin, locale: 'en') }
-    let_it_be(:initiatives) { create_list(:initiative, 3) }
+    let_it_be(:initiatives) { create_list(:initiative, 3, voting_started_at: Time.now) }
```

Comment:
We have to provide a date here, as the related mjml file, `/your_proposed_initiatives_digest_mailer/campaign_mail.mjml`, needs to derive a date from the `expired_at` date, which is calculated based on the `voting_started_at` date.

Todo: Make sure that this mail campaign filters out initiatives with codes that occur before `proposed` (since these will return `nil` for `expired_at`. Should we include all such initiatives in this mail, or only `proposed` (e.g. not `expired`, etc.).

Reply:
Indeed, `campaigns/your_proposed_initiatives_digest.rb` is only sent to authors of initiatives which are `proposed`:
```Ruby
def filter_authors_of_proposed_initiatives(users_scope, _options = {})
   users_scope.where(id: Initiative.published.proposed.select(:author_id))
end
```
So, `voting_started_at` should be set for these initiatives and thus `expires_at` dates can be derived.